src/stage1/errmsg.cpp

```diff
@@ -16,7 +16,7 @@ enum ErrType {
 };
 
 static void print_err_msg_type(ErrorMsg *err, ErrColor color, ErrType err_type) {
-    bool is_tty = os_stderr_tty();
+    bool is_tty = os_stderr_tty() && !os_is_dumb_term();
```

Comment:
I'm nitpicking here but `os_is_dumb_term` should affect `use_colors` and not `is_tty`.